crates/subspace-networking/src/constructor.rs

```diff
@@ -334,7 +334,7 @@ where
                 .expect("Default config for gossipsub is always correct; qed")
         });
 
-        let protocol_version = format!("/subspace/{}", protocol_version);
+        let protocol_version = format!("/subspace-v2/{}", protocol_version);
```

Comment:
It's likely that `/subspace/v2/` is easier to parse for the future logic change.